image upload doesn't work correctly in Webyep 2.1.1 with PHP 7.3

himbeerrot opened the pull request:
Thanks a lot for porting WebYep to PHP7.x!
There is one problem: if I upload an image it doesn't appear on the page.
It's loaded up to /webyep-system/data, but it is not shown on the page.

Commit:
Updated Readme file

Commit:
Update to read me file

Commit:
WebYep2 release

Commit:
small typo updates

Commit:
Inclusion link to Will Woodgate Rapidweaver stacks

maxfancourt committed:
Add files via upload

Commit:
Updated 'manually inserted' loop controls code snippet

Commit:
Updated 'manually inserted' loop controls code snippet

Commit:
WebYep 2.1 release

Commit:
Small description update

Commit:
Updated readme file with 2.1 release notes added

Commit:
Readme file with 2.1 release notes updated

Commit:
The Deprecated: Function get_magic_quotes_runtime() has now been disabled on any PHP higher than 5.4. This was causing an error in WebYep attachments.

maxfancourt committed:
Update README.md

maxfancourt committed:
Update README.md

maxfancourt commented:
> Thanks a lot for porting WebYep to PHP7.x!
> There is one problem: if I upload an image it doesn't appear on the page.
> It's loaded up to /webyep-system/data, but it is not shown on the page.

We haven't had any issues with image file uploads so did you want to contact me directly then I can try to help 
max.izzat(at)gmail.com

maxfancourt committed:
Update README.md

maxfancourt committed:
Update README.md

maxfancourt committed:
Update README.md

maxfancourt committed:
Update README.md

Commit:
Update to Readme file

maxfancourt committed:
Add files via upload

maxfancourt committed:
Update README.md

maxfancourt committed:
Merge branch 'master' of https://github.com/maxfancourt/WebYep

maxfancourt committed:
Update NEW-code-snippets.php

maxfancourt committed:
New Documentation

All new Documentation

maxfancourt committed:
Version 3.0.3 Release

WY 3 - PHP7 and PHP 8 Compatible version of WebYep

maxfancourt committed:
Update README.md

maxfancourt committed:
Update README.md

maxfancourt committed:
Update README.md

maxfancourt committed:
Small Update to readme file

maxfancourt committed:
WebYep 3.0.6 - updates and bug fixes

Longtext tables are now fixed
popup rather than Modal window control is now fixed
WebP file format now recognised in WebYep images and WebYep Gallery
multiple samll errors fixed

maxfancourt committed:
Update README.md

maxfancourt committed:
Update README.md

maxfancourt committed:
Update README.md

maxfancourt committed:
Update NEW-code-snippets.php

New WebYep Audio code snippet and Guest Book code snippet added

maxfancourt committed:
Create webyep-guestbook-example.php

Example of how to integrate the WebYep Guestbook item within a web page

maxfancourt committed:
WebYep 3.1 Codebase Update

This update upgrades the WebYep CMS to version 3.1, including changes to core files for improved compatibility, performance, and stability. Legacy code has been reviewed and updated where necessary to align with the latest version's structure and functionality.

maxfancourt committed:
Update README.md

maxfancourt committed:
Update README.md
